egs2/TEMPLATE/asr1/pyscripts/utils/format_score_scp.py

```diff
@@ -40,7 +40,8 @@ def main():
                 if len(line) == 0:
                     continue
                 utt_id, _, _, _ = line.strip().split(" ")
-                fscp.write(f"{utt_id} {dic[utt_id]}\n")
+                if utt_id in dic.keys():
+                    fscp.write(f"{utt_id} {dic[utt_id]}\n")
```

Comment:
Will this change hurt the other recipes? We'd better have a double check.

Reply:
The change is also unnecessary. Reverted the change.